bin/metal

```diff
@@ -90,17 +90,9 @@ EOF
     #printf "    %-20s %s" "command" "XXX Command synopsis."
     echo ""
 }
-export -f display_help
 
-export alces_BINNAME="metal"
-export alces_BASE="/opt/metalware"
-export alces_CMDDIR="$alces_BASE/lib/actions"
-### RUBY
-export alces_RUBY="$alces_BASE/opt/ruby/bin/ruby"
-export alces_RUBYHOME="$alces_BASE/opt/ruby"
-export LD_LIBRARY_PATH="${alces_RUBYHOME}/lib:$LD_LIBRARY_PATH"
-export GEM_HOME="${alces_RUBYHOME}/lib/ruby/gems/2.1.0"
-export GEM_PATH="${alces_RUBYHOME}/lib/ruby/gems/2.1.0"
+export -f display_help
+$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/exportCmd
```

Comment:
I'm surprised that `bin/exportCmd` would work when executed in this way -- my expectation would be that the `export` statements in there would only affect the execution of that script, not the calling script.

I see that the motivation for this change is to allow both `metal` and `test` to share the same set of variables.  A more conventional pattern for this would be to move the variables to, say `etc/metal.rc` and then `source` that file, something like this:

```
source $(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/../etc/metal.rc
```